modules/serverless-rn-1-4-0.adoc

```diff
@@ -5,6 +5,11 @@
 [id="serverless-rn-1-4-0_{context}"]
 = Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.4.0
 
+[IMPORTANT]
+====
+{ServerlessProductName} 1.4.0 contains a bad owner reference that causes the Kubernetes Garbage Collector to wrongly remove the entire Knative control-plane, including all of your services. You must install {ServerlessProductName} 1.4.1 to fix this issue.
```

Comment:
s/control-plane/control plane ? 
Do we need the hyphen here?

Although grammatically correct, I think, _wrongly remove_ doesn't sound right. Would _incorrectly remove_ be better?